src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @include animation-hover(transform, scale(1.2));
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), mixins should be used for media queries, placeholders, and functions. Using mixins for simple transitions is not recommended as it can make the code less readable due to abstraction. Consider using the transition property directly.